webrtc.html

```diff
@@ -2251,6 +2251,10 @@
             state. Note the local or remote SDP descriptions in the
 	    previous stable state could be null if there has
             not yet been a successful offer-answer negotiation.</p>
+
+            <p>If the value of a <code>RTCRtpTransceiver.mid</code> was set to a non-null value by
+              an SDP description that is rolled back, the <code>RTCRtpTransceiver.mid</code> will be
+              set to null, as defined by <span data-jsep="rollback">[[!JSEP]]</span>.</p>
```

Comment:
I don't think setting a remote description can change the .mid, except between null and a value.  I don't think it can change it from A to B.  Of course, we haven't written that part of JSEP yet, so I may be wrong.